app/main/suppliers.py

```diff
@@ -63,7 +63,7 @@ def suppliers_list_by_prefix():
     page = process_page(request.args.get('page', default=u"1"))
 
     try:
-        api_result = data_api_client.find_suppliers(api_prefix, page, 'gcloud')
+        api_result = data_api_client.find_suppliers(api_prefix, page, 'g-cloud')
```

Comment:
Should we be properly paramaterize these supplier listing pages with `<framework>` in the route while we're here?  Or we don't know if we're going to want these pages for DOS at all yet? (From discussion so far it sounds like we will want something that allows people to see the suppliers on DOS).